cornellh4i/ithaca-recovery
# Implemented view meeting details side bar

Description: Implement the side bar that views meeting details and allows users to edit and remove events. All of the details of the events are props, so they can be reused to suit any meeting with varying details. 

Testing: 
- Navigate to http://localhost:3000/test
- Scroll to the bottom of the page to see a dummy instance of the side bar with random information. Any changes to the instance of ViewMeetingDetails to a different time, date, location should be reflected in the test page. 

Image of components: 

![image](https://github.com/user-attachments/assets/05b3e38e-436e-498e-be27-6c22632c5069)


Notes: 
The width of this organism is a dependent on the width of the screen, but that should be standardize once it is implemented into the page structure. 

Closes #67
